<launch>
	<node pkg="dataAssoc_experimental" type="dataAssoc" name="dataAssoc" output="screen">
		<remap from="pedestrian_clusters" to="vehicle_clusters_filtered"/>
		<remap from="pedestrian_roi" to="vehicle_roi"/>
		<remap from="ped_data_assoc" to="vehicle_data_assoc"/>
		<remap from="ped_data_assoc_visual" to="vehicle_data_assoc_visual"/>
	    <param name="global_frame" value="odom"/>
	    <param name="time_out" value="3.0"/>
	    <param name="poll_increment" value="0.1"/>
	    <param name="poll_decrement" value="0.05"/>
	    <param name="confirm_threshold" value="0.3"/>
	    <param name="image_transport" value="compressed"/>
	</node>
</launch>
